src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    public Figure getRandomFigure(int maxSize) {
+        Random rand = new Random();
```

Comment:
The variable 'rand' should be a class-level field rather than a local variable in the method since it's used across multiple method calls and does not need to be instantiated each time the method is called.